### ISSUE TITLE
No Drone and Track options

### ISSUE THREAD

**Issue by A-Lep** [timower/KwadSim#23]
There are no Drone options and no track options when I run the simulator.

**Comment by timower** [comment]
Weird, at least 4 drones should be available and one track (garageTest).
What platform are you running on?
And can you confirm the contents of Data/quads.json and Data/Tracks/

**Comment by A-Lep** [comment]
yep, they're there. Running on MacOS and Windows, the same issue on both.

**Comment by A-Lep** [comment]
I still dont have them yet in game. They are in the folders but not in game.

**Comment by timower** [comment]
Can you try again with the latest master?
If you still see no drones or tracks please send me the console logs or the logs in `user://Logs/`
